cmd/booster-http/run.go

```diff
@@ -133,37 +119,8 @@ var runCmd = &cli.Command{
 			log.Info("Tracing exporter enabled")
 		}
 
-		maddr, err := storageService.ActorAddress(ctx)
```

Comment:
I moved a bunch of code from booster-http into a shared library (shared with booster-bitswap)